Mage.Sets/src/mage/sets/shadowsoverinnistrad/InfectiousCurse.java

```diff
@@ -0,0 +1,174 @@
+ *  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ *  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ *  SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ *  ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ *  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+ *  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *  The views and conclusions contained in the software and documentation are those of the
+ *  authors and should not be interpreted as representing official policies, either expressed
+ *  or implied, of BetaSteward_at_googlemail.com.
+ */
+package mage.sets.shadowsoverinnistrad;
+
+import java.util.UUID;
+import mage.abilities.Ability;
+import mage.abilities.SpellAbility;
+import mage.abilities.TriggeredAbilityImpl;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.common.AttachEffect;
+import mage.abilities.effects.common.GainLifeEffect;
+import mage.abilities.effects.common.LoseLifeTargetEffect;
+import mage.abilities.effects.common.cost.CostModificationEffectImpl;
+import mage.abilities.keyword.EnchantAbility;
+import mage.cards.CardImpl;
+import mage.constants.CardType;
+import mage.constants.CostModificationType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.Rarity;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.Target;
+import mage.target.TargetPlayer;
+import mage.target.targetpointer.FixedTarget;
+import mage.util.CardUtil;
+
+/**
+ *
+ * @author halljared
+ */
+public class InfectiousCurse extends CardImpl {
+
+    public InfectiousCurse(UUID ownerId) {
+        super(ownerId, 97, "Infectious Curse", Rarity.UNCOMMON, new CardType[]{CardType.ENCHANTMENT}, "");
+        this.expansionSetCode = "SOI";
+        this.subtype.add("Aura");
+        this.subtype.add("Curse");
+
+        this.nightCard = true;
+        this.canTransform = true;
+
+        // Enchant player
+        TargetPlayer auraTarget = new TargetPlayer();
+        this.getSpellAbility().addTarget(auraTarget);
+        this.getSpellAbility().addEffect(new AttachEffect(Outcome.Damage));
+        this.addAbility(new EnchantAbility(auraTarget.getTargetName()));
+
+        // Spells you cast that target enchanted player cost {1} less to cast.
+        this.addAbility(new SimpleStaticAbility(Zone.BATTLEFIELD, new InfectiousCurseCostReductionEffect()));
+        // At the beginning of enchanted player's upkeep, that player loses 1 life and you gain 1 life.
+        InfectiousCurseAbility curseAbility = new InfectiousCurseAbility();
+        curseAbility.addEffect(new GainLifeEffect(1));
```

Comment:
Am I missing something, or is this effect not making the enchanted player lose a life? Looks like it only has the gain life part of the effect here. Didn't see it in InfectiousCurseAbility either.